src/main/java/com/faforever/moderatorclient/ui/ViewHelper.java

```diff
@@ -695,6 +678,13 @@ public static void buildModTableView(TableView<ModFX> tableView, ObservableList<
         tableView.getColumns().add(authorColumn);
         extractors.put(authorColumn, modFX -> modFX.getUploader().getLogin());
 
+        TableColumn<ModFX, Boolean> recommendColumn = new TableColumn<>("Recommended");
+        recommendColumn.setCellValueFactory(o -> o.getValue().recommendedProperty());
+        recommendColumn.setCellFactory(CheckBoxTableCell.forTableColumn(recommendColumn));
+        recommendColumn.setMinWidth(100);
+        tableView.getColumns().add(recommendColumn);
+        extractors.put(recommendColumn, ModFX::isRecommended);
```

Comment:
I don't think we want to offer copying boolean values via context menu. Please remove the extractors.put